src/Microsoft.DocAsCode.Dfm/MarkdownValidators/MarkdownValidationRule.cs

```diff
@@ -9,8 +9,14 @@ namespace Microsoft.DocAsCode.Dfm.MarkdownValidators
 
     public class MarkdownValidationRule
     {
+        /// <summary>
+        /// The contract name of rule.
+        /// </summary>
         [JsonProperty("name", Required = Required.Always)]
```

Comment:
keep compatible with old `md.style`

---
In reply to: [73125280](https://github.com/dotnet/docfx/pull/545#discussion_r73125280) [](ancestors = 73125280)